new_client/src/components/auth/Signin.jsx

```diff
@@ -58,15 +59,17 @@ function Signin(props) {
 	useEffect(() => {
 		if (error.id === LOGIN_FAIL) {
 			setMsg(error.msg.msg);
+			alert(msg);
```

Comment:
Is this alert intentional or was it just for debugging purpose?

Reply:
It was for debugging purposes, I'll remove that part in the next commit.